dox/externalproject/CMakeLists.txt

```diff
@@ -1,29 +1,27 @@
 # Example project for using elastix code from external projects.
-PROJECT( elxExternalProject )
+project( elxExternalProject )
 
-CMAKE_MINIMUM_REQUIRED( VERSION 2.8 )
+cmake_minimum_required( VERSION 3.10.2 )
 
-# Find TIK
-FIND_PACKAGE( ITK REQUIRED )
-IF( ITK_FOUND )
-  INCLUDE( ${ITK_USE_FILE} )
-ENDIF( ITK_FOUND )
+find_package( Elastix REQUIRED )
 
-# find elastix
-SET( ELASTIX_BINARY_DIR "" CACHE PATH "Path to elastix binary folder" )
-SET( ELASTIX_USE_FILE ${ELASTIX_BINARY_DIR}/UseElastix.cmake )
-IF( EXISTS ${ELASTIX_USE_FILE} )
- MESSAGE( STATUS "Including Elastix settings." )
- INCLUDE( ${ELASTIX_USE_FILE} )
-ENDIF()
+# Elastix might use OpenMP.
+find_package( OpenMP QUIET )
+if( OPENMP_FOUND )
+  set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}" )
+endif()
```

Comment:
This was one of the hardest parts for me to figure out: _if_ elastix might use OpenMP, the openmp flag must be added to the compiler flags of the external project! 

On the CI, OpenMP is "found" (by CMake find_package) on Windows and Ubuntu, but not on  macOS.